pegin/getaddress.go

```diff
@@ -0,0 +1,142 @@
+package pegin
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+
+	"github.com/btcsuite/btcutil"
+
+	"github.com/btcsuite/btcd/txscript"
+
+	"github.com/vulpemventures/go-elements/network"
+	"github.com/vulpemventures/go-elements/payment"
+
+	"github.com/btcsuite/btcd/chaincfg"
+
+	"github.com/vulpemventures/go-elements/address"
+
+	"github.com/btcsuite/btcd/btcec"
+)
+
+const (
```

Comment:
woudl be more self describing using string instead of integers here?

Reply:
I think that variable name is already self explanatory and its exported to it can be passed as arg to func GetAddressInfo